### PR TITLE
fix(components/datetime): datepicker emits touched status when focus leaves composite control

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**
[AB#3141597](https://dev.azure.com/blackbaud/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/3141597)

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/2909/storybook/)

Component Storybooks:

- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/2909/storybooks/ag-grid-storybook/)
- [datetime-storybook](https://blackbaud.github.io/skyux-pr-preview/2909/storybooks/datetime-storybook/)
- [lists-storybook](https://blackbaud.github.io/skyux-pr-preview/2909/storybooks/lists-storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/2909/storybooks/tabs-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/2909/code-examples/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/2909/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/2909/playground/)

File: libs/components/datetime/src/lib/modules/datepicker/datepicker-host.service.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import { DestroyRef, Injectable, OnDestroy, inject } from '@angular/core';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+
+import { Observable, Subject, fromEvent } from 'rxjs';
+
+import { SkyDatepickerComponent } from './datepicker.component';
+
+/**
+ * Handles interactions between the colorpicker host component and the input
```
Comment (johnhwhite) [inline libs/components/datetime/src/lib/modules/datepicker/datepicker-host.service.ts]: Fix the comment.

**Review by johnhwhite — COMMENTED** [review]
One small thing. Nice job.

File: libs/components/datetime/src/lib/modules/datepicker/datepicker-host.service.ts (hunk 1 above)
Comment (Blackbaud-SteveBrush) [reply libs/components/datetime/src/lib/modules/datepicker/datepicker-host.service.ts]: Doh! Thanks.

**Review by johnhwhite — APPROVED** [review]

**Pull request merged**